scoper.inc.php

```diff
@@ -0,0 +1,15 @@
+<?php declare(strict_types=1);
+
+return [
+    'prefix' => null,
+    'finders' => [],
+    'patchers' => [],
+    'whitelist' => [
+        'Psr\Container\*',
+        'PHPUnit\Framework\*',
+        'Symfony\Component\Config\*',
```

Comment:
could be simplified to `Symfony\Component\*` and I think you can simply remove the finders & patchers entries